cmd/lakefs/cmd/flare.go

```diff
@@ -0,0 +1,105 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/jedib0t/go-pretty/v6/text"
+	"github.com/spf13/cobra"
+	"github.com/treeverse/lakefs/pkg/flare"
+)
+
+const (
+	flareOutputDirPrefix       = "%s/flare/%s/"
+	flareConfigFileName        = "lakefs-config.yaml"
+	flareDefaultEnvVarFileName = "lakefs-env.txt"
+	flareDefaultZipFileName    = "lakefs-flare.zip"
+	flareDefaultOutputPath     = "."
+)
+
+var (
+	packageContents      bool   = false
+	includeEnvVars       bool   = true
+	outputPath           string = flareDefaultOutputPath
+	envVarOutputFileName string = flareDefaultEnvVarFileName
+	zipOutputFileName    string = flareDefaultZipFileName
+	outputStdout         bool   = false
+)
+
+var flareCmd = &cobra.Command{
+	Use:    "flare",
+	Short:  "collect configuration, environment variables, and logs for debugging and troubleshooting",
+	Args:   cobra.ExactArgs(0),
+	PreRun: warnOutputFlags,
+	Run: func(cmd *cobra.Command, args []string) {
+		flare.SetBaselinePermissions(flare.FlareUmask)
```

Comment:
Instead of all the `flare/permissions_<OS>.go` duplicates can you just add here: 

```
import "runtime"

if runtime.GOOS != GOOSWindows {
		syscall.Umask(mask)
}
```

Reply:
Done. I'm fine either way. My thinking was that we might have other differences between GOOSs as more customers use it, but I guess we can always add them at that point.